### ISSUE TITLE
Add support for self hosted videos

### ISSUE THREAD

**Issue by iwkse** [fabiorino/crelly-slider#55]
Having vimeo and youtube is quite great but sometimes you want more control over the look&feel of the video or you just want to host a video on your own server.

**Comment by fabiorino** [comment]
This is a function I plan to support, but it's not a priority for now, I'm sorry. You wrote me a long email asking for some information about a possible implementation. I replied with some tips.

**Comment by iwkse** [comment]
Yeah thank you. I've just read your email, I'll see if I can provide a pull request on this before you go on implementing it :-)
I really like supporting youtube and video but for some use cases this is not feasible as you know having full control of the iframe is a bit uncomfortable if not even impossible to achieve in certain conditions.

**Comment by Mikeacall** [comment]
Any chance we i could get any of those tips  @fabiorino ?

**Comment by applibs** [comment]
It is ready?

**Comment by chanx** [comment]
any update on this? can we use self hosted video on our media folder?

**Comment by fabiorino** [comment]
It's not possible yet, and I'm afraid I'm not planning to work on it anytime soon due to lack of time. If you're up for the challenge, you're welcome to contribute to the project and I'll review your code.
